src/app/recomendacion-tip/recomendacion-tip.page.ts

```diff
@@ -14,38 +15,38 @@ export class RecomendacionTipPage implements OnInit {
   user: UsuarioGeneral;
 
   constructor(
-    private tipsService: TipsService
+    private tipsService: TipsService,
+    private loginService: LoginService,
   ) { }
 
   ngOnInit() {
     this.obtenerTipRecomendado();
```

Comment:
tal vez deberia poner para que this.user = this.loginService.getUser() solo se haga una vez en el constructor, ya que lo llama varias veces y siempre lo guarda en this.user